### DISCUSSION TITLE
A question about Intersystem bias

### DISCUSSION THREAD

**Discussion by LNTU-YS** [groops-devs/groops#265] [General]
Hello,

During the POD, I found that the results using GPS/Galileo are between only-GPS and only-Galileo. Later, I realized the 'Intersystem bias' (ISB). What should I do if I want to make corrections to it? I have noticed 'GnssBiasClockAlignment' now, but I am not sure if this is correct.

Thank you in advance for your feedback and help.
With Kind Regards
YUAN

**Comment by tmayerguerr** [comment]
Hi Yuan,

Good question. We determine our own GNSS constellation in which we consistently determine the signal bias. In GROOPs, the concept of intersystem biases does not exist, only observable-specific biases (OSB) are calculated. I am not sure how to deal with external products, e.g. from IGS, where not all biases are published. It may actually be possible to solve the problem with GnssBiasClockAlignment.

best regards
Torsten
